resources/views/pages/home-member.blade.php

```diff
@@ -48,55 +18,46 @@
 				Ga je weer een keer mee op kamp?
 			@endif
 		</p>
-		@if ($today)
+		@if ($birthdays->isNotEmpty())
 			<hr/>
 			<h3>Vandaag jarig:</h3>
 			<ul style="font-size:120%;">
-				@foreach ($today as $member)
-					<li>{{$member->voornaam}} {{$member->tussenvoegsel}} {{$member->achternaam}} ({{$member->leeftijd}}) @if ($member->ikjarig) <strong>&larr; dat ben jij! Gefeliciteerd!</strong> <span class="glyphicon glyphicon-gift"></span> @endif
+				@foreach ($birthdays as $m)
+					<li>
+						{{$m->volnaam}} ({{$m->geboortedatum->age}}) 
+						@if (Auth::user()->profile == $m) <strong>&larr; dat ben jij! Gefeliciteerd!</strong> <span class="glyphicon glyphicon-gift"></span> @endif
```

Comment:
Can you multiline this (line break after the if, and before the endif) it makes it way  better readable.